### PR TITLE
Update openjdk/README.md

### PR DESCRIPTION
**Pull request by yukiyuli**
Replace adoptopenjdk.net with adoptium.net #4034 
Add jtreg doc link in openjdk/README.md #4033

### CODE REVIEW

**Comment by llxia** [comment]
@yukiyuli thanks for your PR. Please update your PR title and description to reflect the change in your PR. Please refer [Commit message example](https://github.com/adoptium/aqa-tests/blob/master/Contributing.md) for details. Thanks.

**Comment by yukiyuli** [comment]
> @yukiyuli thanks for your PR. Please update your PR title and description to reflect the change in your PR. Please refer [Commit message example](https://github.com/adoptium/aqa-tests/blob/master/Contributing.md) for details. Thanks.

**Comment by yukiyuli** [comment]
@llxia  I tried to make some changes for commit changes. Please advise if it works.

**Comment by llxia** [comment]
Thanks @yukiyuli  for the update. A couple of issues:
- Please update PR title and description:
image
- Please remove the previously merged commit from this PR https://github.com/adoptium/aqa-tests/pull/4029

**Comment by yukiyuli** [comment]
Hello @llxia , I think that I fixed all issues.

File: openjdk/README.md [hunk 1]
```diff
@@ -14,14 +14,14 @@ See the License for the specific language governing permissions and
 # OpenJDK regression tests
 This group of tests are the set that comes from the openjdk project, often referred to as jtreg tests, as the jtreg framework is the underlying executable used to execute them.  The entire set of openjdk regression tests is quite large.  For our nightly builds, we run only a subset of these tests (typically, those suites of tests that we 'tag' as sanity in the [playlist.xml](https://github.com/adoptium/aqa-tests/blob/master/openjdk/playlist.xml) file).  For release builds, we intend to run the suites tagged as sanity and extended in the playlist.  
 
-For more details on how the underlying jtreg harness works, you can refer to the ["How to Use the JTreg harness" guide](https://adoptopenjdk.gitbooks.io/adoptopenjdk-getting-started-kit/en/intermediate-steps/how_to_use_jtreg_-_java_regression_test_harness.html).  
+For more details on how the underlying jtreg harness works, you can refer to the ["How to Use the JTreg harness" guide](https://openjdk.org/jtreg/command-help.html).
```
Comment (llxia) [inline openjdk/README.md:17]: We should add jtreg doc link, not replace the original `"How to Use the JTreg harness" guide` link.

**Review by llxia — APPROVED** [review]
LGTM

**Review by LongyuZhang — APPROVED** [review]
LGTM

**Pull request merged**